### PR TITLE
[UI] next/prev icon in light theme

### PR DESCRIPTION
**Pull request by aryamanpuri**

### CODE REVIEW

**Comment by aryamanpuri** [comment]
@helfi92

File: ui/src/components/TaskRunsCard/index.jsx [hunk 1]
```diff
@@ -103,6 +103,19 @@ const DOTS_VARIANT_LIMIT = 5;
       marginLeft: theme.spacing.unit / 2,
       marginBottom: theme.spacing.unit / 2,
     },
+    previousPageArrow: {
+      marginLeft: -theme.spacing.unit,
```
Comment (helfi92) [inline ui/src/components/TaskRunsCard/index.jsx]: `theme.spacing.unit` is now `theme.spacing(1)` after the recent material-ui upgrade.

**Comment by aryamanpuri** [comment]
@helfi92  I have updated the pr with the required changes.

**Comment by aryamanpuri** [comment]
@helfi92 Is there any need of the change/modification in the code or is it okay to be merged ?

**Comment by aryamanpuri** [comment]
@helfi92 Any changes required ?

**Pull request merged**